etna/transforms/math/limit.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+import numpy as np
+import pandas as pd
+
+from etna.transforms.base import ReversibleTransform
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders.
+
+    For more details visit https://datasciencestunt.com/time-series-forecasting-within-limits/ .
+    """
+
+    def __init__(self, in_column: str, lower_bound: float = -1e10, upper_bound: float = 1e10):
+        """
+        Init LimitTransform.
+
+        Parameters
+        ----------
+        in_column:
+            column to apply transform
+        lower_bound:
+            lower bound for the value of the column
+        upper_bound:
+            upper bound for the value of the column
+        """
+        super().__init__(required_features=[in_column])
+        self.in_column = in_column
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.tol = 1e-10
+
+    def _fit(self, df: pd.DataFrame):
+        """Fit method does nothing and is kept for compatibility.
+
+        Parameters
+        ----------
+        df:
+            dataframe with data.
+        """
+        pass
+
+    def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Apply scaled logit transform to the dataset.
+
+        .. math::
+            y = \\log (\\frac{x-a}{b-x}),
+
+        where :math:`x` is feature, :math:`a` is lower bound, :math:`b` is upper bound.
+        Parameters
+        ----------
+        df:
+            dataframe with data to transform.
+
+        Returns
+        -------
+        :
+            transformed dataframe
+
+        Raises
```

Comment:
Let's move it to the constructor, this is a correct place, but you won't see it in docs